CHANGES

```diff
@@ -1,3 +1,7 @@
+* Version 1.2.17
+    * fix parallel key-pattern command line usage
+    * fix and clean up build on macOS
```

Comment:
Should be "OSX" or perhaps "Darwin"

Reply:
Not a Mac enthusiast but didn't Apple go from OS X to macOS a few years ago?

Reply:
Not a Mac enthusiast as well, I don't know.